### ISSUE TITLE
NPM package missing

### ISSUE THREAD

**Issue by eshohet** [3box/3box-pinning-node#278]
Error An unexpected error occurred: "https://registry.yarnpkg.com/3box-pinning-node: Not found".

**Comment by oed** [comment]
We haven't actually created an npm package for this repository. Are you looking to use it within another project?

**Comment by eshohet** [comment]
Yes, we'd like to be able to quickly reach peers and fetch pinned content stored via 3box spaces

**Comment by zachferland** [comment]
Are you looking to interact with spaces from a user perspective or query data for any space we pin on our node? then probably looking for 3box-js https://github.com/3box/3box-js (3boxjs does run an ipfs instance in browser and pin user data)

Of do you want to run a node that will pin data for a space for a bunch of users?

**Comment by eshohet** [comment]
We wan to run a node that pins a bunch of our user's data so that we can quickly sync data upon opening our dapp.

**Comment by zachferland** [comment]
@eshohet ok great, just this sprint we were working on bundling this up in an easy and configurable way to pin data for a set of users and then spaces. we can ping you when ready.  The easiest way to run everything will be in https://github.com/3box/3box-services-box, can take a look in the meantime. 

Not sure if will speed up data syncs for you, how slow are things for you and whats the default use case look like? we are generally working to improve that as well
